browser/ui/views/tabs/tab_drag_controller.cc

```diff
@@ -187,9 +187,9 @@ void TabDragController::DetachAndAttachToNewContext(
     std::vector<tabs::TabHandle> tabs;
     auto* tab_strip_model = browser->tab_strip_model();
     DCHECK_EQ(tab_strip_model, attached_context_->GetTabStripModel());
-    for (const auto& tab_drag_data : drag_data_) {
+    for (size_t i = first_tab_index(); i < drag_data_.size(); ++i) {
```

Comment:
This here should be something like:

```cxx
auto drag_data = base::span(drag_data_).subspan(first_tab_index());
for (const auto& tab_drag_data : drag_data) {
  ...
```

This way we don't access the subscript directly.